### DISCUSSION TITLE
Can I split scATAC-seq data for FindTransferAnchors step?

### DISCUSSION THREAD

**Discussion by BuilderPie** [stuart-lab/signac#1262] [Q&A] [answered]
Hi! I had difficulties running the **FindTransferAnchors** step when I was trying to identify anchors between my scRNA-seq and scATAC-seq datasets. The process didn't finish after two days' running, and the peak memory usage was > 300 GB. 

Then I tried to evenly and randomly split the scATAC-seq datasets into **10 subsets**, and ran the **FindTransferAnchors** step on each subset of the scATAC-seq dataset with the **same scRNA-seq dataset as reference**. 

My question is can I combine the resulting 10 transfer.anchors objects to create a unified celltype.predictions object for the following steps, such as annotation and co-embedding of the scRNA and scATAC datasets? 

Thanks for your help!

**Comment by HaikuoLi** [comment]
I also have the simialr question. Is it possible to merge several transfer.anchors (FindTransferAnchors outputs) and then use the merged anchor for data co-embedding?
Hope anyone in the community could help with this question!

**Comment by timoast** [comment, marked as answer]
Running on split objects will not be equivalent to running with all the cells. However, you may find downsampling the cells can give similar results even though you'll have less anchors.

Since this is related to Seurat integration, please ask in the Seurat repo for more information: https://github.com/satijalab/seurat

**Reply by BuilderPie** [reply]
Thank you Tim for your answer and suggestions! I will ask this question in the Seurat repo then.
